website/docs/migrations/7-8.md

```diff
@@ -0,0 +1,151 @@
+---
+id: 7-8
+title: Version 7 to 8
+---
+
+This migration guide is represents migration for:
+
+- [eightshift/boilerplate](https://github.com/infinum/eightshift-boilerplate/releases/tag/8.0.0) - **7+ --> 8.0.0**
+- [eightshift/frontend-libs](https://github.com/infinum/eightshift-frontend-libs/releases/tag/7.0.0) - **6+ --> 7.0.0**
+- [eightshift/libs](https://github.com/infinum/eightshift-libs/releases/tag/6.0.0) - **5+ --> 6.0.0**
+
+### Required changes:
+
+**Migration time: ~10min.**
+
+1. Update `composer.json` to `"infinum/eightshift-libs": "^6.0"`.
+2. Update `package.json` to the latest versions:
+
+	```json
+	"@eightshift/frontend-libs": "^7.0.0",
+	"husky": "^8.0.1",
+	"webpack": "^5.73.0",
+	"webpack-cli": "^4.10.0"
+	```
+
+3. Check `package.json` for additional packages updates if they are out-of-date like code standards.
+4. Add `.browserslistrc` to the root of you theme/plugin and copy the details form [here](https://github.com/infinum/eightshift-boilerplate/blob/develop/.browserslistrc).
+5. We have removed `normalize` package from the project because it was unmaintained. Search your project and replace:
+
+	```scss
+	@import 'normalize-scss';
+	@include normalize();
+	````
+
+	with this:
+	```scss
+	@import '@eightshift/frontend-libs/styles/scss/normalize';
+	```
+
+6. We removed `@babel/polyfill` package so please remove `require('@babel/polyfill');` from you project or install it manually if you are supporting older browsers.
+7. If your Webpack build outputs warnings like this:
+	> Should not import the named export 'componentJsClass' (imported as 'componentJsClass') from default-exporting module (only default export is available soon).
+
+	you should check you JS files used on the frontend and replace:
+
+	```js
+	import { componentJsClass } from './../manifest.json';
+	```
+
+	with this:
+
+	```js
+	import manifest from './../manifest.json';
+
+	const {
+		componentJsClass,
+	} = manifest;
+	```
+
+### Swiper changes:
+
+**Migration time: ~5min.**
+
+Swiper is not our package but if you are using Swiper in your project and have something like this:
+
+```scss
+@import '~swiper/swiper.scss';
+@import '~swiper/modules/pagination/pagination.scss';
+```
+
+please check Swipers `package.json` in `node_modules/swiper/package.json` and find exports key, find the styles you are using and replace the path for the correct export path. Please note that this export can change depending on the swiper version you are using.
+
+
+### Storybook changes:
+
+**Migration time: ~10min.**
+
+1. Check if the project is using storybook package `@eightshift/storybook` because we changed the release numbers, please check this [link](https://github.com/infinum/eightshift-storybook/releases) for references. Our later versions is `1.4.1`.
```

Comment:
```suggestion
1. Check if the project is using storybook package `@eightshift/storybook` because we changed the release numbers, please check this [link](https://github.com/infinum/eightshift-storybook/releases) for references. Our latest version is `1.4.1`.
```